tests/cases.py

```diff
@@ -2446,5 +2446,6 @@ def test_manual(self):
             alignment_t=alignment_t,
         )
 
+    @abstractmethod
```

Comment:
this doesn't necessarily have to be an abstract method, right? like it can just be an empty one that some subclasses don't override and therefore don't add anything to

Reply:
otherwise leave as it is and we can merge this now